src/main/resources/com/cloudbees/jenkins/plugins/advisor/AdvisorGlobalConfiguration/index.jelly

```diff
@@ -4,134 +4,140 @@
   <l:layout title="${it.actionTitleText}" norefresh="true" permission="${it.ADMINISTRATOR}">
 
     <l:header>
-      <link rel="stylesheet" href="${resURL}/bootstrap/css/bootstrap.min.css" type="text/css"/>
       <link rel="stylesheet" href="${resURL}/css/font-awesome/css/font-awesome.min.css"/>
       <style type="text/css">
-        table {
-        border-collapse: separate;
-        border-spacing: 5px;
+        .jenkins-health-advisor .alert-success {
```

Comment:
```suggestion
        /* TODO remove when baseline newer than 2.245, https://github.com/jenkinsci/jenkins/pull/4842 */
        .jenkins-health-advisor .alert-success {
```

Reply:
JENKINS-62739 Add a TODO to remove the custom style later